### PR TITLE
Fix tabs order

### PR DESCRIPTION
**Pull request by bappi**
Fix https://github.com/pluginever/wc-serial-numbers/issues/342

Also change API > API Toolkit in Tools tab

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ Tags: license manager, license, license number, serial number, activation number
 Requires at least: 5.0
 Tested up to: 6.3
 Requires PHP: 5.6
-Stable tag: trunk
```
Comment (sultann) [inline readme.txt:7]: @bappi Why was this changed?

**Review by sultann — CHANGES_REQUESTED** [review]

**Comment by bappi** [comment]
The reviewers don't allow trunk as the stable tag anymore, and they can just take down the plugin just because of that. So it's just a safety measure, and we need to update it with every new release

File: readme.txt (hunk 1 above)
Comment (bappi) [reply readme.txt:7]: The reviewers don't allow trunk as the stable tag anymore, and they can just take down the plugin just because of that. So it's just a safety measure, and we need to update it with every new release
